deploy/docker/local/docker-compose.yml

```diff
@@ -4,7 +4,6 @@ services:
   prometheus:
     build:
       context: prometheus
```

Comment:
`context` already specify the directory, so no need for `dockerfile: prometheus/Dockerfile`.